### PR TITLE
add sponsors to page and update banner image

### PR DESCRIPTION
**Pull request by JessicaS11**
I added ICESat-2 and AWS as sponsors to the home page, and switched to the IS2 banner image (instead of the generic one). Should I also add NASA?

### CODE REVIEW

**Comment by scottyhq** [comment]
Tests are not running if only `cookiecutter.yaml` because it looks like it's missing from the workflow "paths", @JessicaS11 if you add the file under the paths, workflows should run:

https://github.com/uwhackweek/jupyterbook-template/blob/9ff908c848e71e1ec4d0c37f768e26e772d3bcdb/.github/workflows/test.yaml#L9

https://github.com/ICESAT-2HackWeek/website2022/blob/459ab14c2ed7225d5b9073e1d857b872e05bc567/.github/workflows/qaqc.yaml#L5

**Comment by JessicaS11** [comment]
> Tests are not running if only cookiecutter.yaml because it looks like it's missing from the workflow "paths"

See #5 (which we should squash-merge so we avoid the merge commit), after which we can update this branch.

File: cookiecutter.yaml [hunk 1]
```diff
@@ -31,6 +31,13 @@ sponsors:
   - name: eScience Institute
     website: ''
     logo_url: https://icesat-2hackweek.github.io/assets/images/escience.png
+  - name: NASA ICESat-2 Mission
+    website: 'https://icesat-2.gsfc.nasa.gov/mission'
+    logo_url: https://icesat-2hackweek.github.io/assets/images/ICESat2.png
+  - name: Amazon Web Services
+    website: ''
+    logo_url: https://aws.amazon.com/?nc2=h_lg
```
Comment (scottyhq) [inline cookiecutter.yaml:39]: Looks like this AWS logo link isn't correct.  @aaarendt okay to add the NASA logo too?https://www.nasa.gov/sites/default/files/thumbnails/image/nasa-logo-web-rgb.png

File: cookiecutter.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ about: The ICESat-2 Hackweek is a 5-day collaborative learning experience design
 apply:
   url: '#'
   title: Applications Closed
-hackweek_mission: https://uwhackweek.github.io/hackweeks-as-a-service/mission.html
+hackweek_mission: https://icesat-2.hackweek.io/mission.html
 banner:
   title: The ICESat-2 Hackweek 2022
```
Comment (scottyhq) [inline cookiecutter.yaml:9]:
```suggestion
  title: The ICESat-2 Hackweek
```
(looking at the preview, 2022 is repeated on the next line of the title banner)

**Comment by aaarendt** [comment]
Thanks for doing this! Unfortunately we should probably remove the NASA meatball. At least a few years ago when we were making websites for NASA projects, we were told only nasa.gov sites are allowed to use it. Things might have changed since then...

File: cookiecutter.yaml [hunk 3]
```diff
@@ -31,6 +31,12 @@ sponsors:
   - name: eScience Institute
     website: ''
     logo_url: https://icesat-2hackweek.github.io/assets/images/escience.png
```
Comment (scottyhq) [inline cookiecutter.yaml:33]: Just noticing the eScience is super small compared to the others. Should we aim for a certain size? I guess they don't get automatically scaled. Here are two other options: 
https://escience.washington.edu/wp-content/uploads/2015/10/Logo_eScience-stacked.png
https://escience.washington.edu/wp-content/uploads/2015/10/eScience_Logo_HR.png

**Review by scottyhq — APPROVED** [review]

**Pull request merged**